### PR TITLE
Added media print to forms

### PR DESCRIPTION
**Pull request by MiguelAHPpih**

### CODE REVIEW

**Comment by brandones** [comment]
Can we attach some screenshots?

File: configuration/pih/htmlforms/section-dx.xml [hunk 1]
```diff
@@ -52,7 +49,77 @@
             .inline > span > label, .inline > span > input {
                 float: left;
             }
+            <!--Media print specified for this page-->
 
+            @media print 
+            {
+                .navbar {
+                    display: block;
+                }
```
Comment (brandones) [inline configuration/pih/htmlforms/section-dx.xml:58]: You're welcome to handle this how you like, but you could have much less duplicated code if you moved a lot of this (especially the navbar stuff) into `configuration/pih/styles/global/<whatever>.css`

File: configuration/pih/htmlforms/outpatientConsult.xml [hunk 2]
```diff
@@ -291,45 +318,8 @@
...
-                    jq('#show-less-medications-button').hide();
-                    jq('#show-more-medications-button').show();
-                });
-
-                jq('#disposition select:first').change(function() {
-                    jq('#disposition .field-error').first().html('').hide(); // hide any previous error if disposition is changed
-                });
-
+                setUpExpandableSection('diagnosis');
+                setUpExpandableSection('medication');
```
Comment (brandones) [inline configuration/pih/htmlforms/outpatientConsult.xml:322]: Nice! Glad this is working for you

**Review by brandones — APPROVED** [review]
Looks good, just one suggestion that might help make things easier to maintain for you. And please do attach screenshots when you get the chance.

**Comment by MiguelAHPpih** [comment]
@brandones Here is a pic of what is shown with the media print css
imagen

**Comment by brandones** [comment]
Sweet. Feel free to merge whenever.

**Pull request merged**